corehq/motech/repeaters/tasks.py

```diff
@@ -554,6 +556,18 @@ def _get_lock(self, repeater_id):
         return lock
 
 
+@periodic_task(
+    run_every=crontab(hour='5', minute='0', day_of_month='1'),
+    queue=getattr(settings, 'CELERY_PERIODIC_QUEUE', 'celery'),
+)
+def purge_old_datasourceupdates():
+    table_name_format = f'{DataSourceUpdate.Meta.db_table}_y%Ym%m'
+    max_age = datetime.utcnow() - DataSourceUpdate.MAX_AGE
+    table_name = (max_age - timedelta(days=1)).strftime(table_name_format)
+    with connection.cursor() as cursor:
+        cursor.execute(f'DROP TABLE IF EXISTS {table_name}')
```

Comment:
I didn't think about this on the first pass, but is the idea that it is safe to do this because this is secondary data? Meaning it can be rebuilt from existing form/case data? Just double checking since we are comparing this to a form being the payload for a form repeater, which we obviously do not clean up.